corehq/elastic.py

```diff
@@ -33,6 +37,26 @@
 from six.moves import range
 
 
+class ESJSONSerializer(object):
+    mimetype = 'application/json'
+
+    def loads(self, s):
+        try:
+            return json.loads(s)
+        except (ValueError, TypeError) as e:
+            raise SerializationError(s, e)
+
+    def dumps(self, data):
+        # don't serialize strings
+        if isinstance(data, six.string_types):
+            return data
```

Comment:
Does this mean that a string containing JSON-deserializable content will become a different value when deserialized? 

For example:
```py
>>> ejson = ESJSONSerializer()
>>> ejson.loads(ejson.dumps("{}"))
{}

>>> ejson.loads(ejson.dumps("123"))
123

>>> ejson.loads(ejson.dumps("{name}"))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<stdin>", line 7, in loads
__main__.SerializationError: ('{name}', ValueError('Expecting property name: line 1 column 2 (char 1)',))

>>> json.loads(json.dumps("{name}"))  # Real json dumps/loads for comparison
u'{name}'
```

Does that matter?

Reply:
This is taken from the serializer included in the ES client:  https://github.com/elastic/elasticsearch-py/blob/91cd918477ab0cfcc3a7236b7f651d8254045748/elasticsearch/serializer.py#L40

I'll include a comment to indicate that.